src/group_theory/abelianization.lean

```diff
@@ -63,6 +63,9 @@ def of : G →* abelianization G :=
   map_one' := rfl,
   map_mul' := λ x y, rfl }
 
+@[simp] lemma mk_eq_of {a : G} :
+  quot.mk (@setoid.r _ (quotient_group.left_rel (commutator G))) a = of a := rfl
```

Comment:
Does this work fine as
```suggestion
  quot.mk _ a = of a := rfl
```
Lean ought to infer the `_` for you

Reply:
It does! Thanks